src/vaadin-message.js

```diff
@@ -6,7 +6,12 @@
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import '@polymer/iron-icon/iron-icon.js';
```

Comment:
Please do not use `iron-icon`. We are using a different approach in components.

Reply:
Understood. I'll check what we use in other components.